tests/unit/parallel/algorithms/reduce_.cpp

```diff
@@ -32,7 +32,7 @@ void test_reduce1(ExPolicy policy, IteratorTag)
     std::vector<std::size_t> c(10007);
     std::iota(boost::begin(c), boost::end(c), std::rand());
 
-    std::size_t const val(42);
+    std::size_t val(42);
```

Comment:
Got this strange error from clang 5.0 although the variable is used inside the lambda : `warning: lambda capture 'val' is not required to be captured for this use [-Wunused-lambda-capture]`